cfn-resources/project/cmd/resource/resource.go

```diff
@@ -30,7 +30,7 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	project, _, err := client.Projects.Create(context.Background(), &matlasClient.Project{
 		Name:  *currentModel.Name,
 		OrgID: *currentModel.OrgId,
-	})
+	}, &matlasClient.CreateProjectOptions{})
```

Comment:
this does not seem related to the go upgrade. Is this supposed to be here?

Reply:
this PR includes atlas-go-client library upgrade too, its bumped to v0.18.0